Automattic/wp-calypso
# Gutenboarding: V2 Onboarding

After merging this PR, remember to also merge this PR to enable ab testing on verticals: https://github.com/Automattic/wp-calypso/pull/44256

#### Changes proposed in this Pull Request

This is a feature branch PR, including these PRs that were tested separately:

- https://github.com/Automattic/wp-calypso/pull/44655 remove domain as a step
- https://github.com/Automattic/wp-calypso/pull/44701 add vertical input
- https://github.com/Automattic/wp-calypso/pull/44822 https://github.com/Automattic/wp-calypso/pull/44837 vertical gating with `?vertical` URL arg
- https://github.com/Automattic/wp-calypso/pull/44833 enable launch flow in production and bump AB test slug
- https://github.com/Automattic/wp-calypso/pull/44837 persistent show vertical input

#### Does not include:
- https://github.com/Automattic/wp-calypso/pull/44256 A/B test for vertical
    - A/B test to be merged separately.
- https://github.com/Automattic/wp-calypso/pull/44775 plans grid modification
    - Plans grid modification might be clashing with plans grid in launch flow, so we might need to do some separation work here or roll this only once we have an accordion grid in launch flow.



#### Testing instructions

- Go `/new` and observe new vertical input and no mandatory domain step
- Test domain picker from the header

See more pbAok1-1lr-p2
